app/components/createService/createservice.html

```diff
@@ -445,6 +446,52 @@
           <!-- tab-placement -->
           <div class="tab-pane" id="placement" ng-include="'app/components/createService/includes/placement.html'"></div>
           <!-- !tab-placement -->
+          <!-- tab-resources -->
+          <div class="tab-pane" id="resources">
```

Comment:
Instead of creating a new 'Resources' pane, could you group these options with the placements options in a `Resources & Placements` tab ?